android/src/com/androidAuto/HelloWorldScreen.java

```diff
@@ -0,0 +1,25 @@
+package com.androidAuto;
+
+import androidx.annotation.NonNull;
+import androidx.car.app.CarContext;
+import androidx.car.app.Screen;
+import androidx.car.app.model.Action;
+import androidx.car.app.model.Pane;
+import androidx.car.app.model.PaneTemplate;
+import androidx.car.app.model.Row;
+import androidx.car.app.model.Template;
+
+public class HelloWorldScreen extends Screen {
+    public HelloWorldScreen(@NonNull CarContext carContext) {
+        super(carContext);
+    }
```

Comment:
Our code style uses 2 spaces. There should be a code style file for Android Studio in the repo, please check docs or find it.